modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java

```diff
@@ -478,6 +482,12 @@ protected void activate(BundleContext bundleContext) {
 				_objectDefinitionLocalService, _objectFieldLocalService,
 				_organizationLocalService, _roleLocalService,
 				_userGroupRoleLocalService, _userLocalService));
+		_init(bundleContext);
+	}
+
+	@Deactivate
+	protected void deactivate() {
+		_destroy();
```

Comment:
@rijema please inline. no need to create the `_destroy` method as it's only used once.

```
@Deactivate
protected void deactivate() {
	_serviceTrackerList.close();
}
```

Reply:
Ok